code/ui/blocks/src/components/Story.tsx

```diff
@@ -83,13 +80,7 @@ const InlineStory: FunctionComponent<InlineStoryProps> = (props) => {
         )} { min-height: ${height}; transform: translateZ(0); overflow: auto }`}</style>
       ) : null}
       {showLoader && <StorySkeleton />}
-      <div
-        ref={storyRef}
-        id={`${storyBlockIdFromId(props)}-inner`}
-        data-name={story.name}
-        // eslint-disable-next-line react/no-danger
-        dangerouslySetInnerHTML={{ __html: htmlContents }}
-      />
```

Comment:
@tmeasday This seemed to be an issue. Can you take a look at this? Maybe you remember why the dangerouslySetInnerHTML was necessary at all.

Reply:
@valentinpalkovic I think it is explained by the comment above:

> We do this so React doesn't complain when we replace the span in a secondary render

IIRC, if you render (in an "outer" react context [the docs page in this case]):

```
<div><span /></div>
```

And then later use a second "inner" react context [the react renderer in this case] to overwrite the `span` (`ReactDOM.render(<something />, theSpan`), then React helpfully warns you that you are overwritting one render tree with another. This was a trick to avoid that.

It may longer be an issue or there may be a better way. I am not sure.

Reply:
Possibly the canary version of react changes this behavior. and we can no longer depend on this trick @tmeasday.

Reply:
If so great!